ZenPacks/zenoss/PythonCollector/zenpython.py

```diff
@@ -332,6 +343,20 @@ def __init__(self, taskName, configId, scheduleIntervalSeconds, taskConfig):
         self.metricExtraTags = getattr(
             self._dataService, "metricExtraTags", False)
 
+    def getDatasources(self):
+        try:
+            (template, datasource) = self.choosenDatasource.split('/')
+        except ValueError:
+            log.error('Invalid datasource format')
+            return []
+        filteredDatasources = [
+            ds for ds in self.config.datasources
+            if ds.template == template and ds.datasource == datasource]
+        if len(filteredDatasources) == 0:
+            log.error('No configs for template %s, datasource %s',
```

Comment:
Please use the file's existing indentation scheme throughout. This means no aligning indentation with opening parenthesis. There are three other cases of this in the following changes.
```suggestion
            log.error(
                'No configs for template %s, datasource %s',
                template. datasource)
```